src/containers/org/components/RemoveMemberFromOrgModal.js

```diff
@@ -55,6 +65,13 @@ const RemoveMemberFromOrgModal = ({
     [RequestStates.FAILURE]: (
       <ResetOnUnmount actions={RESET_ACTIONS}>
         <Typography>Failed to remove member. Please try again.</Typography>
+        <Typography>
+          {
+            thisIsYou
+              ? 'Failed to leave organization. Please try again.'
+              : 'Failed to remove member. Please try again.'
+          }
+        </Typography>
       </ResetOnUnmount>
     ),
   };
```

Comment:
'Delete Person' sounds scary for the `textTitle` of this modal.

Reply:
I agree. I'll update.